SignalMessaging/utils/UIUtil.m

```diff
@@ -41,6 +41,9 @@ + (void)setupSignalAppearence
     //                                                 ? UIKeyboardAppearanceDark
     //                                                 : UIKeyboardAppearanceDefault);
 
+    //    [[UIBarButtonItem appearanceWhenContainedIn:[UISearchBar class], nil] setTintColor:[UIColor
+    //    ows_materialBlueColor]];
```

Comment:
?

Reply:
Ah, came back with the 'Revert "Add custom themed search bar."'

I'll remove it.